astropy/visualization/lupton_rgb.py

```diff
@@ -267,8 +267,7 @@ def __init__(self, minimum, stretch, Q=8):
             Q = 0.1
         else:
             Qmax = 1e10
-            if Q > Qmax:
-                Q = Qmax
+            Q = min(Q, Qmax)
```

Comment:
Is Qmax even needed? (Could write 1e10 directly in min call?)

Reply:
good call. Done !